test/Microsoft.ML.Tests/PermutationFeatureImportanceTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿//------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Float = System.Single;
```

Comment:
please just use `float` in code, and delete this one.

Reply:
done.